packages/devtools_app/lib/src/shared/diagnostics/inspector_service.dart

```diff
@@ -189,11 +191,12 @@ abstract class InspectorServiceBase extends DisposableController
 /// Manages communication between inspector code running in the Flutter app and
 /// the inspector.
 class InspectorService extends InspectorServiceBase {
-  InspectorService()
+  InspectorService(IsolateRef? evalIsolate)
       : super(
           clientInspectorName: 'WidgetInspectorService',
           serviceExtensionPrefix: inspectorExtensionPrefix,
           inspectorLibraryUri: inspectorLibraryUri,
+          evalIsolate: evalIsolate,
```

Comment:
this can just be `serviceConnection.serviceManager.isolateManager.mainIsolate`

no need to add the evalIsolate parameter to InspectorService

Reply:
🤦 fair enough :P thats what I get for doing this during a meeting :P